docker/user-bazelrc

```diff
@@ -0,0 +1,2 @@
+# user: jenkins-megaboom@foss-fpga-tools-ext-openroad.iam.gserviceaccount.com
```

Comment:
nit: although not a secret, ideally we wouldn't even have to expose this... it is a server configuration detail, not source code.

I'd leave it be unless you have a ready alternative.

Reply:
Vitor and I chatted about this and I'd prefer to make the change within buildMegaboom.groovy using a Jenkins secret instead of DockerHelper.sh. That way dockerPush.groovy doesn't have to change that much.